dymos/phase/phase.py

```diff
@@ -1522,9 +1548,9 @@ def configure(self):
 
         transcription.configure_defects(self)
 
-        transcription.configure_boundary_constraints('initial', self)
+        _configure_constraint_introspection(self)
 
-        transcription.configure_boundary_constraints('final', self)
+        transcription._configure_boundary_constraints(self)
```

Comment:
Kind of wondering why this is the only configure-stack method that has an underscore.

Reply:
I'm trying to be more cognizent of those methods and members which should be underscored.  There's no reason for the user to call the configure methods so they should be underscored, I just haven't moved them all that way yet.